python/lsst/pipe/supertask/super_task.py

```diff
@@ -137,7 +137,7 @@ def print_config(self):
         for testing, it may disappear or be replaced with something else.
         """
         print('\n* Configuration * :\n')
-        for key, val in self.config.items():
+        for key, val in list(self.config.items()):
```

Comment:
My fault, I ran `futurize` on the code which inserted this nonsense and I missed it when reviewed what `futurize` did. Will remove all of that.